webapp/src/containers/modals/LinkModal/index.tsx

```diff
@@ -63,6 +65,9 @@ const LinkModal = () => {
         serviceName: pluginConstants.pluginApiServiceConfigs.createLink.apiServiceName,
         payload: formFields as LinkPayload,
         handleSuccess: () => {
+            if ((data as CreateLinkResponse)?.message === 'This project is already linked.') {
```

Comment:
Create constants for this errors as i think they are coming from the APIs and that can change in the future.